src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -118,7 +118,9 @@ public function paginationAction(Request $request, $limit = 10, $offset = 0, $to
                 unset($callerParameters[$k]);
             }
         }
-        $routeName = $request->attributes->get('caller_route', $request->attributes->get('caller_parameters', ['_route' => false])['_route']);
+        
+        $callerParameters += array('_route' => false);
```

Comment:
Always prefer short array syntax
```suggestion
        $callerParameters += ['_route' => false];
```

Reply:
it won't that's the main difference between + union operator and array_merge:
```
>>> ['a' => 1] + ['a' => 2]
=> [
     "a" => 1,
   ]
```
```
>>> array_merge(['a' => 1], ['a' => 2])
=> [
     "a" => 2,
   ]
```
it does not override the already existing elements of array but will only append them if they don't exist. 
it's virtually the same as you wrote it with array_merge, just need to make sure the order of arguments is correct so the more important one always override the defaults, that's why i chose union

Reply:
Oh my bad, always thought it'll be override :sweat_smile:

Reply:
I see we use array() everywhere, just wanted to be consistent, since it's not enforced by code style rules